src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset class="fieldset">
+        <legend>Personal Information</legend>
+
+        <label class="person-info">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="person-info">
+          Full date of birth:
+          <input
+            type="date"
+            name="bithday"
+          >
+        </label>
+
+        <label class="person-info">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Regestration</legend>
+
+        <label class="person-info">
+         E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="person-info">
+          Password
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="person-info">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              checked
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+            >
+              No
+          </label>
+        </div>
+
+        <label class="person-info">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+
+
+        <label class="person-info">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
```

Comment:
```suggestion
          <input
            type="time"
            name="time"
            step="1"
          >
```
You can add step attribute to choose seconds